_posts/0000-01-02-jlspader1078.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
The past doesn't exist. It is a perception of linear movement attempting to explain one's current state.